### ISSUE TITLE
Address root causes of missing csets

### ISSUE THREAD

**Issue by joeflack4** [jhu-bids/TermHub#573]
## Overview
In #521, we looked into why concept sets are missing from TermHub.

## Sub-tasks list
- [x] 1. New csets (should be covered by db refresh)
- [ ] 2. Old csets (should be covered by db refresh / initialize)

### 1. New csets (should be covered by db refresh)
We came up with a solution. We have a script that will fetch missing csets and a test that will check for them.

I was then going to run the "find-and-add-missing-csets" script right after the refresh. However, that solution would just be a band-aid until we found the root of the problem.

I think the root of the problem is this:
- #574

Basically this was happening, I think, because `refresh_status` was never set from `active` back to `inactive` (probably a debugging issue). For now, this issue is also fixed because I manually changed it. However, #574 will implement an automated solution for that.

If it turns out this doesn't solve the problem, I'll re-open this issue.

### 2. Old csets (should be covered by db refresh / initialize)
Still have not found the issue for this, but I've moved this into a subtask of #399.

**Comment by joeflack4** [comment]
Current thoughts: Root problem?
a. Maybe because Maya just fixed some drafts not available? *
My money on this
b. If drafts get a new cset ID when finalized, that could be it?
c. Sometimes csets delay in when new csets available in API? Becomes available after we've already fetched and marked that DB as current given timestamp of fetch?
